Detectors/TPC/base/include/TPCBase/Defs.h

```diff
@@ -51,26 +51,47 @@ enum class PadSubset : char {
   Partition,  ///< Partitions (up to 36*5)
   Region      ///< Regions (up to 36*10)
 };
-/// Pad centres as 2D float
-typedef Point2D<float> PadCentre;
-typedef Point2D<float> GlobalPosition2D;
-typedef Point2D<float> LocalPosition2D;
-typedef Point3D<float> GlobalPosition3D;
-typedef Point3D<float> LocalPosition3D;
+/// default point definitions
+template <typename T>
+using Point3D     = ROOT::Math::PositionVector3D<ROOT::Math::Cartesian3D<T>, ROOT::Math::DefaultCoordinateSystemTag>;
+template <typename T>
+using Point3Dglob = ROOT::Math::PositionVector3D<ROOT::Math::Cartesian3D<T>, ROOT::Math::GlobalCoordinateSystemTag>;
+template <typename T>
+using Point3Dloc  = ROOT::Math::PositionVector3D<ROOT::Math::Cartesian3D<T>, ROOT::Math::LocalCoordinateSystemTag>;
```

Comment:
How about the 2D definiations:
`template <typename T>
using Point2D     = ROOT::Math::PositionVector2D<ROOT::Math::Cartesian2D<T>, ROOT::Math::DefaultCoordinateSystemTag>;
template <typename T>
using Point2Dglob = ROOT::Math::PositionVector2D<ROOT::Math::Cartesian2D<T>, ROOT::Math::GlobalCoordinateSystemTag>;
template <typename T>
using Point2Dloc  = ROOT::Math::PositionVector2D<ROOT::Math::Cartesian2D<T>, ROOT::Math::LocalCoordinateSystemTag>;
`
Do you want to have a Cartesian2D.h, or shall I leave it in the TPC part?